thundra/opentracing/tracer.py

```diff
@@ -16,20 +16,24 @@
 
 class ThundraTracer(opentracing.Tracer):
     __instance = None
-
-
-    @staticmethod
-    def create_instance(scope_manager=ThreadLocalScopeManager()):
-        return ThundraTracer(scope_manager)
-
+    
 
     @staticmethod
     def get_instance():
         return ThundraTracer() if ThundraTracer.__instance is None else ThundraTracer.__instance
 
 
     def __init__(self, scope_manager=None):
-        scope_manager = ThreadLocalScopeManager() if scope_manager is None else scope_manager
+        import sys
+        if scope_manager is None:
+            print(sys.version_info)
+            if (sys.version_info[0] > 3) or (sys.version_info[0] == 3 and (sys.version_info[1] >= 6 and sys.version_info[2] != 0)):
```

Comment:
Can we move this logic to a utility and put all the logic inside try/catch. In case of, we can continue with the `ThreadLocalScopeManager`